sp800-63b/sec4_aal.md

```diff
@@ -74,37 +74,37 @@ When a combination of two single-factor authenticators is used, it SHALL include
 
-Communication between the claimant and verifier (the primary channel in the case of an out-of-band authenticator) SHALL be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to MitM attacks.
+Communication between the claimant and verifier – the primary channel in the case of an out-of-band authenticator – SHALL be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to MitM attacks.
 
 Verifiers operated by government agencies at AAL2 SHALL be validated to meet the requirements of [FIPS 140](#FIPS140-2) Level 1.
 
-When a device such a smartphone is used in the authentication process, the unlocking of that device (typically done using a PIN or biometric) SHALL NOT be considered one of the authentication factors. This is because it is generally not possible for a verifier to know that the device had been locked, or if the unlock process met the requirements for the relevant authenticator type.
+When a device such a smartphone is used in the authentication process, the unlocking of that device – typically done using a PIN or biometric – SHALL NOT be considered one of the authentication factors. This is because it is generally not possible for a verifier to know that the device had been locked or if the unlock process met the requirements for the relevant authenticator type.
 
-When a biometric factor is used in authentication at AAL2, the performance requirements stated in [Section 5.2.3](#biometric_use) SHALL be met, and the verifier SHOULD make a determination that the biometric sensor and subsequent processing meet these requirements.
+When a biometric factor is used in authentication at AAL2, the performance requirements stated in [Section 5.2.3](#biometric_use) SHALL be met and the verifier SHOULD make a determination that the biometric sensor and subsequent processing meet these requirements.
 
 #### <a name="aal2reauth"></a>4.2.3 Reauthentication
 
-Periodic reauthentication of subscriber sessions SHALL be performed as described in [Section 7.2](#sessionreauthn). At AAL2, authentication of the subscriber SHALL be repeated at least once per 12 hours during an extended usage session, regardless of user activity. Reauthentication of the subscriber SHALL be repeated following any period of no more than 30 minutes of session inactivity. The session SHALL be terminated (i.e., logged out) when either of these time limits is reached.
+Periodic reauthentication of subscriber sessions SHALL be performed as described in [Section 7.2](#sessionreauthn). At AAL2, authentication of the subscriber SHALL be repeated at least once per 12 hours during an extended usage session, regardless of user activity. At a minimum, reauthentication of the subscriber SHALL be repeated following any period of inactivity lasting 30 minutes. The session SHALL be terminated (i.e., logged out) when either of these time limits is reached.
```

Comment:
I almost used the exact language that Jim suggested but I was trying to maintain the original "no more than" concept from the original. His changes sound much better.